include/ck/utility/generic_memory_space_atomic.hpp

```diff
@@ -0,0 +1,97 @@
+#pragma once
```

Comment:
Pleause use 

```
git mv include/ck/utility/generic_memory_space_atomic_add.hpp include/ck/utility/generic_memory_space_atomic.hpp
```
instead, to let git know this is a file renaming instead of a new file, this preserve the commit history

Reply:
Actually, I use vscode git extension to commit the code. I will pay attention on this.